lib/alloc/malloc.h

```diff
@@ -9,10 +9,19 @@
 #include <config.h>
 
 #include <stdlib.h>
+#include <limits.h>
 
 #include "attr.h"
 
 
+#ifndef PATH_MAX
+#define PATH_MAX 4096
+#endif
+
+#ifndef MAXHOSTNAMELEN
+#define MAXHOSTNAMELEN 64
+#endif
+
```

Comment:
Thanks!  This is nicer.

But please move it to "lib/defines.h", since "lib/alloc/malloc.h" isn't very appropriate for these values.

Reply:
moved so